src/meta_schedule/schedule_rule/schedule_rule.cc

```diff
@@ -85,6 +85,39 @@ Array<ScheduleRule> ScheduleRule::DefaultLLVM() {
   };
 }
 
+Array<ScheduleRule> ScheduleRule::DefaultVNNI() {
+  return {
+      ScheduleRule::AutoInline(
```

Comment:
We recently added new schedule rules `ApplyCustomRule` and `InlineConstantScalars` for llvm, let's also add them here for consistency